package/gluon-l3roamd/files/etc/modules.d/gluon-l3roamd

```diff
@@ -0,0 +1 @@
+cfg80211
```

Comment:
If mmfd does not work without *cfg80211* loaded, such a file should rather be part of the l3raomd package.

Does mmfd do anything useful with *cfg80211* when there are no WLAN adapters? I think it would be much better to fix mmfd to work correctly without *cfg80211* adding a hack to load the module.